README.rst

```diff
@@ -0,0 +1,70 @@
+Image Modal XBlock
+==================
+A fullscreen image modal XBlock.
+
+This package is meant to be installed within the OpenEdX platform.
+
+.. image:: ./docs/lms-view-normal.png
+
+
+Installation, system administrator
+----------------------------------
+
+To install the XBlock on your platform,
+add the following to your `requirements.txt` file:
+
+    xblock-image-modal
+
+You'll also need to add this to your `INSTALLED_APPS`:
+
+    imagemodal
+
+
+Installation, course staff
+--------------------------
+
+To install the XBlock in your course,
+access your `Advanced Module List`:
+
+    Settings -> Advanced Settings -> Advanced Module List
+
+.. image:: ./docs/settings-menu.png
+
+and add the following:
+
+    imagemodal
+
+.. image:: ./docs/advanced-module-list.png
+
+
+Use, Course Staff
```

Comment:
Right now, I'm not taking time to explain how to add a component to your outline;
is that a fine omission or should it be included?

Reply:
That seems fine to omit to me.